### DISCUSSION TITLE
Does anybody else's Colab session crash with the Tutorials?

### DISCUSSION THREAD

**Discussion by Soffia-t** [ourownstory/neural_prophet#1398] [General]
Everytime I try to use the tutorials on Colab (yes, the official ones on Neural Prophet's Github) my session crashes. According to Colab it is a RAM problem, but I am running on Colab Pro, it crashes every single time. Anybody have a solution?

**Comment by leoniewgnr** [comment]
Discussion is continued here: https://github.com/ourownstory/neural_prophet/issues/1399
